### ISSUE TITLE
libraries/libgd: build fails with recent libvpx installed on the system

### ISSUE THREAD

**Issue by ackalker** [part-cw/lambdanative#65]
On Arch Linux x86_64, with package libvpx 1.4.0-2 installed, I'm getting the
following error when building in LambdaNative:

```
webpimg.c: In function 'VPXEncode':
webpimg.c:714:24: error: 'IMG_FMT_I420' undeclared (first use in this function)
     vpx_img_wrap(&img, IMG_FMT_I420,
                        ^
webpimg.c:714:24: note: each undeclared identifier is reported only once for each function it appears in
webpimg.c:716:16: error: 'PLANE_Y' undeclared (first use in this function)
     img.planes[PLANE_Y] = (uint8*)(Y);
                ^
webpimg.c:717:16: error: 'PLANE_U' undeclared (first use in this function)
     img.planes[PLANE_U] = (uint8*)(U);
                ^
webpimg.c:718:16: error: 'PLANE_V' undeclared (first use in this function)
     img.planes[PLANE_V] = (uint8*)(V);
                ^
Makefile:772: recipe for target 'webpimg.lo' failed
make[1]: *** [webpimg.lo] Error 1
```

I found that libvpx upstream removed the legacy compatibility layer defining
these constants in [this commit](https://chromium.googlesource.com/webm/libvpx/+/0ecc75c81986a277d2e8e387117edfc21cb2fed0..9cdaa3d72eade9ad162ef8f78a93bd8f85c6de10/vpx/vpx_image.h). They need to be prefixed now.

There exists a [fix](https://github.com/gagern/libgd/commit/d41eb72cd4545c394578332e5c102dee69e02ee8), which I will convert into a patch suitable for LambdaNative
and make a pull request.

Note that this patch will probably need to be revisited when upstream releases a new
stable version of libgd, because libvpx [may](https://bitbucket.org/libgd/gd-libgd/issues/129/) be [dropped](https://bitbucket.org/libgd/gd-libgd/commits/a79232c5fa69) in favor of libwebp.

**Comment by ackalker** [comment]
Made a pull request: #66

**Comment by clpetersen** [comment]
I've merged your pull request in commit e58382a, thanks!

**Comment by ackalker** [comment]
On 08/10/2015 05:41 PM, Chris Petersen wrote:
> I've merged your pull request in commit e58382a
> <https://github.com/part-cw/lambdanative/commit/e58382a18f1150c57027a3558e0cfca4ad461739>,
> thanks!

You're welcome :)
